src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigrateBuildOptionsRule.cs

```diff
@@ -104,6 +104,11 @@ private AddPropertyTransform<CommonCompilerOptions>[] KeyFileTransforms
                 @"$(OutputPath)\$(TargetFramework)\$(AssemblyName).xml",
                 compilerOptions => compilerOptions.GenerateXmlDocumentation != null && compilerOptions.GenerateXmlDocumentation.Value);
 
+        private AddPropertyTransform<CommonCompilerOptions> AssemblyNameTransform =>
+            new AddPropertyTransform<CommonCompilerOptions>("AssemblyName",
```

Comment:
NIT: put the "AssemblyName" parameter in the next line, so that all parameters are in their own line. Easier to read that way.